src/managing-deis/platform-ssl.md

```diff
@@ -1,3 +1,58 @@
 # Platform SSL
 
-TODO (bacongobbler): rewrite for v2
+SSL/TLS is the standard security technology for establishing an encrypted link between a web server
+and a browser. This link ensures that all data passed between the web server and browsers remain
+private and integral.
+
+To enable SSL for your cluster and all apps running upon it, you can add an SSL key to your load
+balancer. You must either provide an SSL certificate that was registered with a CA or provide
+[your own self-signed SSL certificate](../reference-guide/creating_self_signed_ssl.md).
+
+## Installing SSL on a Load Balancer
+
+On most cloud-based load balancers, you can install a SSL certificate onto the load balancer
+itself. Any communication inbound to the cluster will be encrypted while the internal components
+of Deis will still communicate over HTTP.
+
+To enable SSL, you will need to open port 443 on the load balancer and forward it to port 80 on the
+routers. For AWS, you'll also need to add port 443 in the security group settings for your load
+balancer.
+
```

Comment:
> These are directions to modify the load balancer

I figured.

> I am assuming at this point after "installing Deis", we've already provisioned a kubernetes cluster that has a listener at port 80.

If running on AWS (which is where these docs are most applicable, because GKE does things differently) and k8s provisions the load balancer automatically, then there would also already be a listener on port 443-- although it would be connected to the high-numbered port on the kube-proxy that connects the the router's service's 443.  This would also be using TCP and not HTTPS.  (This is just how k8s would have set it up.)  So modifying the load balancer to terminate SSL is really a matter of:

1.  Swap the listener on 443 to use HTTPS instead of TCP.
2.  Swap the back end of that listener to use HTTP _and_ whatever back end port the listener on 80 is using.
3.  Install the cert.